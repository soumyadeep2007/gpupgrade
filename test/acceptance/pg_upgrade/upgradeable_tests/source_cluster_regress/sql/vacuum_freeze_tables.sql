-- Copyright (c) 2017-2021 VMware, Inc. or its affiliates
-- SPDX-License-Identifier: Apache-2.0

-- Test to ensure that frozen tables can be upgraded and the

--------------------------------------------------------------------------------
-- Create and setup upgradeable objects
--------------------------------------------------------------------------------
CREATE TABLE vf_tbl_heap (a int, b int);
INSERT INTO vf_tbl_heap SELECT i, i FROM GENERATE_SERIES(1,10)i;
VACUUM FREEZE vf_tbl_heap;

CREATE TABLE vf_tbl_ao (a int, b int) WITH (appendonly=true);
CREATE INDEX vf_tbl_ao_idx1 ON vf_tbl_ao(b);
INSERT INTO vf_tbl_ao SELECT i, i FROM GENERATE_SERIES(1,10)i;
VACUUM FREEZE vf_tbl_ao;

CREATE TABLE vf_tbl_aoco (a int, b int) WITH (appendonly=true, orientation=column);
CREATE INDEX vf_tbl_aoco_idx1 ON vf_tbl_aoco(b);
INSERT INTO vf_tbl_aoco SELECT i, i FROM GENERATE_SERIES(1,10)i;
VACUUM FREEZE vf_tbl_aoco;
