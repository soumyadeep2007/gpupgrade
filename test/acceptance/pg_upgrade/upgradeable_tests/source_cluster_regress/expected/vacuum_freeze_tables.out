-- Copyright (c) 2017-2021 VMware, Inc. or its affiliates
-- SPDX-License-Identifier: Apache-2.0

-- Test to ensure that frozen tables can be upgraded and the

--------------------------------------------------------------------------------
-- Create and setup upgradeable objects
--------------------------------------------------------------------------------
CREATE TABLE vf_tbl_heap (a int, b int);
CREATE
INSERT INTO vf_tbl_heap SELECT i, i FROM GENERATE_SERIES(1,10)i;
INSERT 10
VACUUM FREEZE vf_tbl_heap;
VACUUM

CREATE TABLE vf_tbl_ao (a int, b int) WITH (appendonly=true);
CREATE
CREATE INDEX vf_tbl_ao_idx1 ON vf_tbl_ao(b);
CREATE
INSERT INTO vf_tbl_ao SELECT i, i FROM GENERATE_SERIES(1,10)i;
INSERT 10
VACUUM FREEZE vf_tbl_ao;
VACUUM

CREATE TABLE vf_tbl_aoco (a int, b int) WITH (appendonly=true, orientation=column);
CREATE
CREATE INDEX vf_tbl_aoco_idx1 ON vf_tbl_aoco(b);
CREATE
INSERT INTO vf_tbl_aoco SELECT i, i FROM GENERATE_SERIES(1,10)i;
INSERT 10
VACUUM FREEZE vf_tbl_aoco;
VACUUM
